### DISCUSSION TITLE
Follow a book , follow an author

### DISCUSSION THREAD

**Discussion by DevMiMar** [bookwyrm-social/bookwyrm#2806] [Ideas]
Just an idea but I dont know if that is even possible (ActivityPub?) as it's not account related.
I would like
 to follow a book (and followers will get a post if "something" happend with this book...comments..start reading...etc.)
 to follow an author (and followers will get a post if "something" happend with a book related to this author...comments..start reading...etc.)

**Comment by jaschaurbach** [comment]
That'll be great!

**Reply by codeinabox** [reply]
I'd love a feature where I could follow an author and get notifications when they publish a new book,

**Reply by roke-julian-lockhart** [reply]
Currently, Amazon provides that, for reference.
